src/representations/calculator_spherical_expansion.hh

```diff
@@ -849,29 +1156,40 @@ namespace rascal {
       } else {  // Default false (don't compute gradients)
         this->compute_gradients = false;
       }
+
       this->spherical_harmonics.precompute(this->max_angular,
                                            this->compute_gradients);
 
+      // create the class that will compute the radial terms of the
+      // expansion. the atomic smearing is an integral part of the
+      // radial contribution
+      //
+      auto smearing_hypers = hypers.at("gaussian_density").get<json>();
+      auto smearing_type = smearing_hypers.at("type").get<std::string>();
+
+      if (smearing_type.compare("Constant") == 0) {
+        this->atomic_smearing_type = AtomicSmearingType::Constant;
+      } else if (smearing_type.compare("PerSpecies") == 0) {
```

Comment:
why do you check specifically for "PerSpecies" and "Radial" instead of using the catch all `else` clause?

Reply:
because this features are implemented soon